util/src/main/java/tech/pegasys/artemis/util/config/ArtemisConfigurationDeprecated.java

```diff
@@ -0,0 +1,456 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.util.config;
+
+import static java.util.Arrays.asList;
+
+import com.google.common.base.Strings;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.tuple.Pair;
+import org.apache.tuweni.config.Configuration;
+import org.apache.tuweni.config.PropertyValidator;
+import org.apache.tuweni.config.Schema;
+import org.apache.tuweni.config.SchemaBuilder;
+
+/** Configuration of an instance of Artemis. */
+public class ArtemisConfigurationDeprecated {
```

Comment:
We do not need this class. This will be removed in a follow-up PR. I will need a bunch of the tests from `ArtemisConfigurationDeprecatedTest` when I implement some CLI validation logic.